src/exodus/tx.h

```diff
@@ -315,6 +315,8 @@ class CMPTransaction
         if (block != other.block) return block > other.block;
         return tx_idx > other.tx_idx;
     }
+
+    boost::signals2::signal<void (CMPTransaction* const)> NotifyProcessedTransaction;
```

Comment:
This is not right due to it is impossible for subscriber to subscribe all `CMPTransaction`. We should move this to `exodus` namespace and make it global.